### PR TITLE
Rename QTZ_linear_mid_tread_half

### LINKED ISSUE #574: Rename QTZ_binary_channel_wise_mean_scaling and other quantizers
All node.op in tensorflow graph uses camel case. But our customized `QTZ_binary_channel_wise_mean_scaling` and other quantizers use snake case. I think we'd better change them to keep consistency.
Screen Shot 2019-10-30 at 11 26 57

### PR DESCRIPTION
**Pull request by Joeper214**
## Motivation and Context

- Last PR for Issue #574 
- fix #574 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature / Optimization (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: blueoil/converter/plugins/tf.py [hunk 1]
```diff
@@ -90,8 +90,18 @@
...
 }
 
+FLOAT32_TENSOR_TYPES = [
+    'BinaryMeanScalingQuantizer',
+    'LinearMidTreadHalfQuantizer',
+    'BinaryChannelWiseMeanScalingQuantizer',
+    'QTZ_binary_channel_wise_mean_scaling',
+    'QTZ_binary_mean_scaling',
+    'QTZ_linear_mid_tread_half',
+]
```
Comment (tfujiwar) [inline blueoil/converter/plugins/tf.py]: Set literal {a, b, c} would be better for this purpose.

**Comment by iizukak** [comment]
@Joeper214 
Can you change quantizer name in the documentation?
https://docs.blue-oil.org/converter/supported_ops.html

**Comment by Joeper214** [comment]
> @Joeper214
> Can you change quantizer name in the documentation?
> https://docs.blue-oil.org/converter/supported_ops.html

Sure. I'll do that. Thanks for noticing.

**Pull request merged**